go/rtl/auth.go

```diff
@@ -142,7 +142,7 @@ func (s *AuthSession) Do(result interface{}, method, ver, path string, reqPars m
 	}
 
 	// set headers
-	req.Header.Add("Content-Type", contentTypeHeader)
+	req.Header.Set("Content-Type", contentTypeHeader)
 
 	if s.Config.AgentTag != "" {
 		req.Header.Set("User-Agent", s.Config.AgentTag)
```

Comment:
you should also set `x-looker-appid` like the other SDKs do. There are some request routers that will modify the `User-Agent` header. Currently this is set in the existing `auth.go` and the intent is that AgentTag overwrites that default header value.

Reply:
Good callout, was unaware of the behavior of AgentTag in relation to `x-looker-appid`. Will have a separate ticket to tackle that.